modules/data.atmosphere/R/met.process.R

```diff
@@ -452,24 +453,25 @@ browndog.met <- function(browndog, source, site, start_date, end_date, model, di
     logger.warn("Could not process model", model)
     invisible(return(NA))
   }
-  
+
   xmldata <- paste0("<input>",
                     "<type>", source, "</type>",
                     "<site>", sitename, "</site>",
                     "<lat>", site$lat, "</lat>",
                     "<lon>", site$lon, "</lon>",
                     "<start_date>", start_date, "</start_date>",
                     "<end_date>", end_date, "</end_date>",
+                    "<username>", username, "</username>"
```

Comment:
Travis failed because this line was missing a comma at the end. Just pushed a fix